artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -375,11 +376,14 @@ private Scanner getScanner() {
          scanner.setReportDirs(false);
          scanner.setReportExistingFilesOnStartup(false);
          scanner.setScanDepth(1);
-         scanner.addListener((Scanner.BulkListener) filenames -> {
-            for (String filename: filenames) {
-               List<Runnable> tasks = scannerTasks.get(filename);
-               if (tasks != null) {
-                  tasks.forEach(t -> t.run());
+         scanner.addListener(new Scanner.BulkListener() {
+            @Override
+            public void filesChanged(Set<String> filenames) throws Exception {
```

Comment:
It seems filesChanged was deprecated for removal, so perhaps its worth switching to the new pathChanged to avoid a future break?

https://github.com/jetty/jetty.project/blob/jetty-12.1.0/jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java#L330-L386